### DISCUSSION TITLE
Prometheus -> Usage Stats Bridge

### DISCUSSION THREAD

**Discussion by toddtreece** [grafana/grafana#48342] [Platform]
Would it be possible to implement a Prometheus to usage stats bridge similar to the [Graphite](https://github.com/grafana/grafana-enterprise/issues/2648) bridge? The bridge could collect and aggregate usage stats for metrics in the `usage_stats` namespace, and probably would need a way to persist the aggregated value so it's available after restarting Grafana.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
